.changeset/tall-dancers-exercise.md

```diff
@@ -0,0 +1,8 @@
+---
+'@primer/react': patch
+---
+
+- Inner action's hover bg should not touch the text input edges
+- Increases touch target area of the inner action button
+- Deprecation: we only want to allow inner action buttons to be icon buttons
+- Deprecation: we only want to allow inner action buttons to be the 'invisible' variant
```

Comment:
Should we rephrase to look more like a change log? 

"we only want to allow inner action buttons to be icon buttons" sounds a bit rude in a changelog 😅

Reply:
Well that's no good. Changing!